server/routes/testutils/testData.ts

```diff
@@ -489,8 +489,8 @@ export default class TestData {
     },
     prisoner = {
       prisonerNumber: 'A1234BC',
-      firstName: 'JOHN',
-      lastName: 'SMITH',
+      firstName: 'John',
+      lastName: 'Smith',
```

Comment:
```suggestion
      firstName: 'JOHN',
      lastName: 'SMITH',
```

I think this needs to stay capitalised because this is how this endpoint returns this data (for the **prisoner** - visitor names are lower case). If we don't have it in capitals here we could potentially not lower case it and the tests would still pass.